code/obj/storage/crates.dm

```diff
@@ -493,6 +493,13 @@ TYPEINFO(/obj/storage/crate/chest)
 			bux.pixel_x = rand(-9, 9)
 			bux.pixel_y = rand(0, 6)
 
+/obj/storage/crate/mail
+	name = "internals crate"
+	desc = "An internals crate."
```

Comment:
is this the correct name and desc? doesn't seem right.

Reply:
lol whoops it's not, but it gets set in shippingmarket.dm anyway

Reply:
will fix